narwhals/utils.py

```diff
@@ -199,30 +199,33 @@ def namespace(self) -> type[Namespace[Any]]:
 class Implementation(Enum):
     """Implementation of native object (pandas, Polars, PyArrow, ...)."""
 
-    PANDAS = auto()
-    """Pandas implementation."""
-    MODIN = auto()
+    PANDAS = "pandas"
+    """pandas implementation."""
```

Comment:
pandas should be lower case - see [citing, Brand and logo](https://pandas.pydata.org/about/citing.html)